CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - CCF now supports a mode where HTTP redirect responses are returned, rather than relying on internal forwarding. This can be used to ensure that write requests are executed on a primary, even if they are initially sent to a backup. This behaviour is enabled by setting the `redirections` field in an `rpc_interface` within cchost's launch config. This can be configured to redirect either directly to a node (if each node has a distinct, accessible name), or to a static load balancer address, depending on the current deployment. See docs for description of [redirection behaviour](https://microsoft.github.io/CCF/main/architecture/request_flow.html#redirection-flow) and [configuration](https://microsoft.github.io/CCF/main/operations/configuration.html#redirections).
+- Authentication policies can now be conjoined (AND) together, in addition to the previous disjoint (OR) behaviour. The new `ccf::AllOfAuthnPolicy` takes a collection of other policies, _all of which must be true_ for this auth policy to pass. In JS, [this can be configured](https://microsoft.github.io/CCF/main/build_apps/js_app_bundle.html#allofauthnpolicy) in the `app.json` as `"authn_policies": [{ "all_of": ["policy_a", "policy_b"] }]`.
```

Comment:
allOf?

Reply:
The surrounding keys are all `little_snakes`, so I think `allOf` looks out of place?

```
    "/multi_auth": {
      "post": {
        "js_module": "logging.js",
        "js_function": "multi_auth",
        "forwarding_required": "always",
        "authn_policies": [
          {
            "allOf": ["user_cert", "jwt", "user_cose_sign1"]
          },
          "user_cert",
          ...
        ],
        "mode": "readwrite",
        "openapi": {},
        "interpreter_reuse": { "key": "singleton_interpreter" }
      }
```

Reply:
That seems reasonable, in which case I think the .rst needs fixing, that's where I saw "allOf".